profile/README.md

```diff
@@ -12,22 +12,22 @@
 
 # The power of the community
 
-Welcome to charm community!
+Welcome to the Charm Community!
 
 A community-led project that is officially backed by the [charmbracelet][charm] organization.
 
 ## Our projects
 
-- [additional-bubbles][additional-bubbles], a collection of bubbles for the [bubbletea][bubbletea] library.
+- [additional-bubbles][additional-bubbles], a collection of Bubbles for the [Bubble Tea][bubbletea] library.
 - [charm-in-the-wild][charm-in-the-wild], the best of what the CLI has to offer.
 
 ## How can I get involved?
 
-We are always looking for more libraries, bubbles, and other awesome projects to include in our collection.
+We are always looking for more libraries, Bubbles, and other awesome projects to include in our collection.
 
 ### How can I get my library added?
 
-The first question you should ask is "is this better as a bubble?" If the answer is yes, consider adding it to the [additional-bubbles][additional-bubbles].
+The first question you should ask is “is this better as a bubble?” If the answer is yes, consider adding it to the [additional-bubbles][additional-bubbles].
```

Comment:
Do we really have to use `“ ”` quotes? Is it related to the branding? I think normal quotes would be more suitable.